code/modules/mob/living/simple_animal/abnormality/zayin/bead_marker.dm

```diff
@@ -0,0 +1,63 @@
+#define STATUS_EFFECT_BLINDPOINT /datum/status_effect/blindpoint
```

Comment:
preferably, put an empty line between
#define STATUS_EFFECT_BLINDPOINT /datum/status_effect/blindpoint
and
/mob/living/simple_animal/hostile/abnormality/bead_marker